chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -102,8 +102,8 @@ class KryoBase extends Kryo {
 object Instantiators {
   // Go through the list and use the first that works
   def newOrElse(cls: Class[_],
-    it: TraversableOnce[Class[_] => Either[Throwable, ObjectInstantiator]],
-    elsefn: => ObjectInstantiator): ObjectInstantiator = {
+    it: TraversableOnce[Class[_] => Either[Throwable, ObjectInstantiator[AnyRef]]],
```

Comment:
we need to move to `Try[ObjectInstantiator[AnyRef]]` here. Our use of `Either` predates us scala having `Try`.